deathbeds/jupyak
# a11y prs

## what kind of request is this?

- [x] a preview site on ReadTheDocs
  - [x] it only adds a `jupyak_config.toml`
- [ ] a `jupyak` code change
  - [ ] it adds new features, described below

## for preview sites

### how should a visitor use the preview site?



1. open a new **private** browser window
2. go to https://jupyak--16.org.readthedocs.build/en/16/_static/work/lite/lab/
3. see the _Launcher_
3. use <kbd>tab</kbb> one or more times to focus the _Python (pyodide)_ launcher item
4. press <kbd>enter</kbd> to start a new notebook from the launcher
5. press <kbd>esc</kbd> to enter _Command Mode_
6. use <kbd>shift+tab</kbb> a few times navigate back to _Close_ button for the activity
7. use <kbd>delete</kbd> to close the notebook
8. accept the dialog default with <kbd>enter</kbd>

### where is the main discussion?

https://github.com/t03857785/jupyterlab/issues/12
